packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -5938,6 +5938,10 @@ Map {
         ],
         "type": "shape",
       },
+      "id": Object {
+        "isRequired": true,
+        "type": "string",
+      },
```

Comment:
Should we make the id required and make a public API change, or, by default, generate a few characters as a hash as carbon does with other components like the tooltip, and override that with the ID if one is given?

Reply:
Yeah we should not require this and instead allow for the id prop to override the default like Kevin suggest. We do  have uuid package that we use in certain components.